src/components/AssetBreadcrumb/stories/AssetBreadcrumb.stories.tsx

```diff
@@ -30,6 +31,15 @@ class CogniteClient extends MockCogniteClient {
   };
 }
 
+const retrieveAssets = async (assetId: CogniteInternalId): Promise<Asset[]> => {
```

Comment:
so it should be either called `retrieveAsset` and have signature 
`async (assetId: CogniteInternalId): Promise<Asset>`
 or called  `retrieveAssets` and have this signature
`async (assetIds: CogniteInternalId[]): Promise<Asset[]>`